plugin/firewall/README.md

```diff
@@ -0,0 +1,112 @@
+# firewall
+
+## Name
+
+*firewall* - enables filtering on query and response using direct expression as policy.
```

Comment:
'direct expressions' is a bit weird, what do you eamn?

Reply:
ok. changed.